docs/getting-started/summaryofchanges.md

```diff
@@ -135,11 +135,11 @@ The following bugs were fixed.
 - Make use of external certificate authorities referenced during keystore setup time.
 - ZSS startup would issue warnings about failure to write yml files for APIML in the case APIML was not also being used.
 - Bugfix: In previous versions, external certificate authorities were not registered with the app server properly and would sometimes contribute to a SELF_SIGNED_CERT_IN_CHAIN error when using the mediation layer. This issue has been resolved by adding external CA certs to the app-server CA array. [#138](https://app.zenhub.com/workspaces/community-5c93e02fa70b456d35b8f0ed/issues/zowe/zlux-app-server/138)
-- 
+-
```

Comment:
Again, don't think it's that important, but you have an empty bullet point here.

Reply:
Thank you, these are important bc it makes things look messy and/or incomplete if you hadn't noticed!